apycula/gowin_pack.py

```diff
@@ -1962,6 +1962,53 @@ def set_osc_attrs(db, typ, params):
         add_attr_val(db, 'OSC', fin_attrs, attrids.osc_attrids[attr], val)
     return fin_attrs
 
+def bin_str_to_dec(str_val):
+    bin_pattern = r'^[0,1]+'
+    bin_str = re.findall(bin_pattern, str_val)
+    if bin_str:
+        dec_num = int(bin_str[0], 2)
+        return str(dec_num)
+    return None
+    
+
+
+_hclk_default_params ={"GSREN": "false", "DIV_MODE":"2"}
+def set_hclk_attrs(db, params, num, typ, cell_name, device):
```

Comment:
there is no need to pass device here - this is already a global variable

Reply:
That's true! I'll correct that.